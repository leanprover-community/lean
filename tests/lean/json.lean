#eval to_string $ json.parse $ "[0.7]"
#eval to_string $ json.parse $ "{\"hello\": 0.4, \"world\": [null, true, false, 1,2,3, \"this is a \\\" string with an annoying quote in it. \"]}"
#eval to_string $ json.parse $ "spurgles"