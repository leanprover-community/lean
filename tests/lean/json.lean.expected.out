"(some [0.699999988079071])"
"(some {\"hello\":0.400000005960464,\"world\":[\"this is a \\\" string with an annoying quote in it. \",3,2,1,false,true,null]})"
"none"
